terra-jupyter-aou/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.4 - 2021-11-24T05:51:34.197137
```

Comment:
you also need to update version [here](https://github.com/DataBiosphere/terra-docker/blob/master/config/conf.json#L146)

Reply:
ah, thanks! I am about to ask you about this

Reply:
Hello @Qi77Qi can you help to check the new error? I do not know what happend this time. Also, is Leo caching automattly happened after this PR merge? When do I expect that happen? Thanks!

Reply:
there's no error yet....CI is still building.

For caching, another PR needs to happen.. See the diff in 2 `preparexxx` script in https://github.com/DataBiosphere/leonardo/pull/2219/files...do you have access to jenkins?

Reply:
It has an error, and I just return GHA.  Now it failed again. 
I can connect to Broad VPN, so I should have access to jenkins? What I should do for the another PR? Run those two script locally?